### PR TITLE
RHBPMS-4854 - GlobalTimerService.timerJobsPerSession grows without bo…

### PR DESCRIPTION
**Pull request by mswiderski**
…unds

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

File: jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalTimerServiceVolumeTest.java [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+        RuntimeEnvironment environment = RuntimeEnvironmentBuilder.Factory.get()
+                .newDefaultBuilder()
+                .entityManagerFactory(emf)
+                .userGroupCallback(userGroupCallback)
+                .addAsset(ResourceFactory.newClassPathResource("org/jbpm/test/functional/timer/IntermediateCatchEventTimerCycleWithHT3.bpmn2"), ResourceType.BPMN2)
+                .schedulerService(globalScheduler)
+                .registerableItemsFactory(new TestRegisterableItemsFactory(listener, countDownListener))
+                .get();
+
+        if (strategy == 1) {
```
Comment (rsynek) [inline jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalTimerServiceVolumeTest.java]: I think this test could be designed much cleaner as a parameterized test, putting the code above into a `@Before` method and making the `manager` instance variable - the test parameter.

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins execute full downstream build

**Pull request merged**